src/Content/Widget.php

```diff
@@ -598,4 +598,9 @@ public static function channels(string $base, string $channelname, int $uid): st
 			$channelname
 		);
 	}
+
+	public static function visibleForUser(int $uid, string $channel): bool
```

Comment:
This doesn't look relevant to this PR.

Reply:
Yeah, a leftover. I thought I had removed it.